src/language-js/parse/postprocess/index.js

```diff
@@ -108,10 +137,13 @@ function postprocess(ast, options) {
   });
 
   if (isNonEmptyArray(ast.comments)) {
-    let followingComment = ast.comments.at(-1);
-    for (let i = ast.comments.length - 2; i >= 0; i--) {
+    let followingComment;
+    for (let i = ast.comments.length - 1; i >= 0; i--) {
       const comment = ast.comments[i];
+      assertComment(comment, text);
```

Comment:
I think assertions for production environments should be considered separately from assertions for development environments.
Therefore, I propose to add a simple validation like the one below and remove the relevant part from `assertComment` and wrap `assertComment` call in `if (process.env.NODE_ENV !== "production")`:

```js
if (!isLineComment(comment) && !isBlockComment(comment)) {
    throw new TypeError(`Unknown comment type: "${comment.type}".`);
}
```

Reply:
Make sense.